tools/api-designer/package.json

```
@@ -11,7 +11,10 @@
     "bootstrap": "^5.0.2",
     "highlight.js": "^11.1.0",
     "odata-openapi": "^0.8.0",
+    "odata-url-editor": "file:../urlEditor",
```

According to `README.md` there should be a dependency to `odata-uri`:

```suggestion
    "odata-url-editor": "file:../urlEditor",
    "odata-uri": "file:../odataUri",    
```

---

The instructions are misleading, this doesn't need to be referenced as it doesn't get used directly but the `api-designer`. It's already referenced by the `odata-url-editor`.